modules/samples-operator-overview.adoc

```diff
@@ -4,30 +4,29 @@
 
 
 [id="samples-operator-overview_{context}"]
-= Understanding the Samples Operator
+= Understanding the Cluster Samples Operator
 
 During installation, the Operator creates the default configuration object for
 itself and then creates the sample imagestreams and templates, including
 quickstart templates.
 
-The Samples Operator copies the pull secret that is captured by the installation
+The Cluster Samples Operator copies the pull secret that is captured by the installation
 program into the OpenShift namespace and names the secret,
 `samples-registry-credentials`, to facilitate imagestream imports from
 `registry.redhat.io`. Additionally, to facilitate imagestream imports from other
```

Comment:
`The Cluster Samples Operator copies the pull secret that is captured by the installation program into the OpenShift namespace and names the secret, samples-registry-credentials, to facilitate imagestream imports from registry.redhat.io`  need removed.